ctf/org.eclipse.tracecompass.ctf.core/src/org/eclipse/tracecompass/ctf/core/event/types/IntegerDeclaration.java

```diff
@@ -438,6 +504,27 @@ private void setVarint(boolean varint) {
         fVarint = varint;
     }
 
+    /**
+     * Getter for mappings
+     *
+     * @return a mapped range of integers
+     * @since 4.5
+     */
+    public Map<String, List<IntegerRange>> getMappings() {
+        return fMappings;
+    }
+
+    /**
+     * Setter for mappings
+     *
+     * @param mappings
+     *            a mapped range of integers
+     * @since 4.5
+     */
+    public void setMappings(Map<String, List<IntegerRange>> mappings) {
```

Comment:
Can this be private?

Reply:
You're right, it should be private.
Fixed 👍